armada/handlers/k8s.py

```diff
@@ -58,3 +58,10 @@ def get_namespace_pod(self, namespace="default"):
         '''
 
         return self.client.list_namespaced_pod(namespace)
+
+    def get_all_pods(self):
+        '''
```

Comment:
use """ not '''

write using active voice 3rd person omitted present

`Returns a list of pods from across all namespaces`